client/src/cmdhfmfdes.c

```diff
@@ -6504,6 +6504,179 @@ static int CmdHF14ADesDeleteFile(const char *Cmd) {
     return PM3_SUCCESS;
 }
 
+static int CmdHF14ADesValueOperations(const char *Cmd) {
+    CLIParserContext *ctx;
+    CLIParserInit(&ctx, "hf mfdes value",
+                  "Get File Settings from file from application. Master key needs to be provided or flag --no-auth set (depend on cards settings).",
+                  "hf mfdes value --aid 123456 --fid 01 --op credit -d 00000001 -> change file settings app=123456, file=01 with defaults from `default` command\n"
+                  "hf mfdes value -n 0 -t des -k 0000000000000000 -f none --aid 123456 --fid 01 -rawdata 00EEEE -> execute with default factory setup\n"
```

Comment:
looks complex.  Why not `--reset`   to reset back to default values?

Reply:
There are some bugs...  Thank you)
If you want - you can specify connection parameters there. if not - it will use defaults